### ISSUE TITLE
Question: best way to install the latest extended version

### ISSUE THREAD

**Issue by jbsilva** [NeoHsu/asdf-hugo#7]
Hi. Thanks for this plugin!

To install the latest extended version I'm currently running the following:

`asdf install hugo extended_$(asdf list-all hugo | tail -1)`

Is there a better way to do it?

I would love to have something like `asdf install hugo latest:extended`.

**Comment by mdgreenwald** [comment]
While this syntax works for the extended releases, there doesn't seem to be any syntax that can handle the new ` `extended_withdeploy_` prefix.

hugo = 'extended_0.146.5'

**Comment by jbsilva** [comment]
I imagine that replacing `tail -1` with a `grep` would work.
But I don't use this plugin anymore as I don't need different versions of Hugo.

If you need to install the latest `extended_withdeploy` from the command line, you can start with something like the following:

```bash
curl -sL https://api.github.com/repos/gohugoio/hugo/releases/latest | \
  jq -r '.assets[] | select(.name | test("^hugo_extended_withdeploy_.*_linux-amd64\\.deb$")).browser_download_url'
```

**Comment by mdgreenwald** [comment]
Yeah I hear you. I'd prefer the plugin work again for asdf/mise but yes there is always an option to fallback to downloading manually.

**Comment by pboling** [comment]
Currently, it works for vanilla hugo only, it seems.

**Comment by aaccioly** [comment]
I know it has been a while since this was requested, but I need the `extended_withdeploy` version for one of my projects and I'm willing to contribute this. I also have a potential fix for both the original issue (i.e., installing with `asdf install hugo latest:extended`) and for `mise-en-place` (`mise use -g hugo@extended_withdeploy`).

In theory, appending the `extended_withdeploy` and `extended` versions (in that order) to the list command should be enough, as both asdf and mise will fetch the latest version starting with those respective classifiers. I can put a PR together if anyone is still interested.

**Comment by NeoHsu** [comment]
Sorry for taking so long to address this.

This is now supported on `master` in `99b7f0b`.

For asdf, you can now install the latest extended Hugo variants with:

```shell
asdf install hugo latest:extended
asdf install hugo latest:extended_withdeploy
```

The plugin resolves those aliases to the latest available release variants, currently:

```text
latest:extended            -> extended-0.162.0
latest:extended_withdeploy -> extended_withdeploy-0.162.0
```

I also verified both variants with `asdf plugin test` against the updated plugin. Closing this now, but please reopen or file a new issue if you still hit problems.
